media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-r1 python-utils-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine."
+HOMEPAGE="http://www.yafaray.org"
+SRC_URI="https://github.com/YafaRay/Core/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/YafaRay/Blender-Exporter/archive/v${PV}.tar.gz -> ${PN}-blender-exporter-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+blender debug +fastmath +fasttrig jpeg opencv openexr png +python -ruby tiff truetype"
+
+RDEPEND="dev-libs/boost
+	dev-libs/libxml2
+	sys-libs/zlib
+	blender? ( media-gfx/blender )
+	jpeg? ( virtual/jpeg:0 )
+	opencv? ( >=media-libs/opencv-3.1.0[openexr?] )
+	openexr? ( >=media-libs/openexr-2.2.0 )
+	png? ( media-libs/libpng:= )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( dev-lang/ruby:= )
+	tiff? ( media-libs/tiff:0 )
+	truetype? ( media-libs/freetype )"
+DEPEND="${RDEPEND}
+	dev-lang/swig"
+
+S="${WORKDIR}/Core-${PV}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i "s/-Wall/${CXXFLAGS}/g" CMakeLists.txt # respect flags
+
+	if use blender; then
+		pushd "${WORKDIR}/Blender-Exporter-${PV}"
+		epatch "${FILESDIR}/${P}-blender-exporter-paths.patch"
```

Comment:
No `epatch` in EAPI 6.

Reply:
Looks like Herb needs to bookmark EAPI 6. Fixed.